packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -332,7 +334,22 @@ def setup_vip_workload_tests(dcos_api_session, container, vip_net, proxy_net, ip
     return tests
 
 
+def create_app_prefix(vip_net. proxy_net, container, ipv6, named_vip):
+    prefixes = [
+        'test-vip',
+        str(vip_net),
+        str(proxy_net),
+        str(container),
+        'ipv6' if ipv6 else 'ipv4'
+        ]
+    if named_vip:
+        prefixes.append('namedvip')
+
+    return '-'.join(prefixes)
+
+
 def vip_workload_test(dcos_api_session, container, vip_net, proxy_net, ipv6, named_vip, same_host):
+    app_prefix = create_app_prefix(vip_net, proxy_net, container, ipv6, named_vip)
```

Comment:
I like this change very much! However, we would get extremely long names. Even if we use `.value.lower()` it would be 93-char string, e.g. `integration-test-test-vip-user-user-docker-ipv6-namedvip-proxy-9003a0924e8d4807b0ae05dcac8bb13d`. DNS have a strict limitation, it's 64 octets for labels. I would suggest to make a prefix from `named_vip` and `host`, all other information is already logged well by pytest.

Reply:
Which `host` parameter do you mean?

Reply:
I meant `same_host`. if it's false we launch proxy app on second slave, but we can easily get this information from app `ip`.